app/routers/quizzes.py

```diff
@@ -20,7 +21,9 @@ async def create_quiz(quiz: Quiz):
 
         client.quiz.questions.insert_many(questions)
 
-    return JSONResponse(status_code=status.HTTP_201_CREATED, content=created_quiz)
+    return JSONResponse(
+        status_code=status.HTTP_201_CREATED, content={"quiz_id": new_quiz.inserted_id}
+    )
```

Comment:
@suryabulusu I still feel like you haven't totally understood the purpose and usage of pydantic models
please read this well https://pydantic-docs.helpmanual.io/ 
also the models section here https://pydantic-docs.helpmanual.io/usage/models/

Or I might be wrong and you know the usage of pydantic models, but your solution is not a scalable one.
What if we need another field in the response? You'll have to add them here itself manually.
And if you remove the pydantic models, you will not get the helpful features like smart type conversion and all.
For example in this case, if you had used a response model, pydantic would automatically check if the inserted_id was a string or not, None or not and throw the exception on its own.
But if you're not using pedantic models, you'll have to write those error handlings yourself.

Reply:
Also, if some other dev wanted to know ki what is this function returning, they would have to comb through the code and look at the last line where you're returning JSONResponse.
Whereas in the rest of the code, all the outputs and inputs are defined as pydantic models, so everything is in one place and super easy to digest